app/api/orders.py

```diff
@@ -71,11 +71,11 @@ def is_payment_valid(order, mode):
 def check_billing_info(data):
     if data.get('amount') and data.get('amount') > 0 and not data.get('is_billing_enabled'):
         raise UnprocessableEntity({'pointer': '/data/attributes/is_billing_enabled'},
-                                  "Billing information is mandatory for paid orders")
-    if data.get('is_billing_enabled') and not (data.get('company') and data.get('address') and data.get('city') and
-                                               data.get('zipcode') and data.get('country')):
+                                  "Billing information is mandatory for this order.")
+    if data.get('is_billing_enabled') and not (data.get('company') and data.get('address')
+	                and data.get('city') and data.get('zipcode') and data.get('country')):
```

Comment:
Why? Is it relevant to this issue?

Reply:
This are the mandatory checks for billing details hence I have merged the two as suggested .

Reply:
@kushthedude Ok. Change PR title.